easybuild/easyconfigs/r/R-bundle/pbdBASE-02.3-0_ignore-BI_COMM_GLOBAL-on-intel.patch

```diff
@@ -0,0 +1,45 @@
+# Add support for intel compilers by removing references to BI_COMM_WORLD
```

Comment:
it's actually "add support for Intel MPI" (and patch name should be changed too, to mention `intel-mpi` or `impi`

and I think we should just remove the lines, rather than guard them by `ifndef __INTEL_COMPILER`

if someone builds pbdBASE with GCC + Intel MPI, this patch would be needed to, but wouldn't work as is